File: __tests__/__fixtures__/output/node/request/query.js
const request = require('request');

const options = {
  method: 'GET',
  url: 'https://httpbin.org/anything',
  qs: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'}
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/java/nethttp/fixtures/query.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value"))
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());